### PR TITLE
Remove D2K Music mirrors (for the time being)

### PR DESCRIPTION
**Pull request by jaZzKCS**
[10:36]	penev|m:	We should take down those d2k music packages

[11:13]	jaZz_KCS:  Well, in the client the options to DL the music package is greyed out no matter how many mirrors there actually are. But I agree, we shouldn't even be hosting the music files on the mirrors... I suggest emptying the mirrors file for the d2k music data, until the (c) situation is made explicitly clear. Until then, no direct connection between OpenRA and obtainable music packages for this mod.

[11:14]	penev|work: exactly

### CODE REVIEW

**Comment by jaZzKCS** [comment]
Also this is referring to: https://github.com/OpenRA/OpenRA/pull/9162

**Comment by LipkeGu** [comment]
#9162 adds the function to download the music... (the line is enabling this) for your intention... (c) situation is releated to all mods! in this case

**Comment by jaZzKCS** [comment]
LipkeGu:

This is not about the fact that you are able to downlaod the music with the ingame button. We were talking abouth the fact that we are - in contrast to CNC and RA -  not distributing the d2k music due to copyright reasons.

Therefore there shall be no direct connection between OpenRA and the music packs, hence the mirror link shall be removed from the txt file for the time being, as discussed on IRC.

File: content/packages/d2k-music-mirrors.txt [hunk 1]
```diff
@@ -1 +1 @@
-http://81.169.246.181/d2k-music.zip
+
```
Comment (Mailaender) [inline content/packages/d2k-music-mirrors.txt]: I suggest you delete the whole file.

**Comment by jaZzKCS** [comment]
Mailaender added a note 2 days ago

I suggest you delete the whole file.


-- done.

**Comment by jaZzKCS** [comment]
It looks like we still have a Travis bubu...

**Comment by Mailaender** [comment]
Can you squash the commits?

**Comment by Mailaender** [comment]
> It looks like we still have a Travis bubu...

https://github.com/OpenRA/OpenRAWeb/pull/275 should fix it.

**Comment by pchote** [comment]
Superseded by #285.

**Pull request closed without merging**